docs/caveat-unique-values.doctest.rst.inc

```diff
@@ -0,0 +1,121 @@
+.. _caveat-unique-values:
+
+Values Must Be Unique
+---------------------
+
+Consider the following::
+
+    >>> from bidict import bidict
+    >>> b = bidict({0: 0, 1: 1})
+    >>> b[0] = 1  # doctest: +SKIP
+
+What should happen next?
+
+If the bidict allowed this to succeed,
+because of the uniqueness-of-values constraint,
+it would silently clobber both of the existing mappings,
```

Comment:
Good point. The latest version of the docs (which has your other recommendations) should improve this.